apps/AppWithWearable/lib/backend/api_requests/stream_api_response.dart

```diff
@@ -99,29 +104,39 @@ String? jsonEncodeString(String? regularString) {
   return encodedString.substring(1, encodedString.length - 1);
 }
 
-void handlePartialResponseContent(String data, Future<dynamic> Function(String) callback) {
+void handlePartialResponseContent(
+    String data, Future<dynamic> Function(String) callback) {
   if (data.contains("content")) {
-    ContentResponse contentResponse = ContentResponse.fromJson(jsonDecode(data));
+    ContentResponse contentResponse =
+        ContentResponse.fromJson(jsonDecode(data));
     if (contentResponse.choices != null &&
         contentResponse.choices![0].delta != null &&
         contentResponse.choices![0].delta!.content != null) {
-      String content = jsonEncodeString(contentResponse.choices![0].delta!.content!)!;
+      String content =
+          jsonEncodeString(contentResponse.choices![0].delta!.content!)!;
       callback(content);
     }
   }
 }
 
-String qaStreamedBody(String context, List<Message> chatHistory) {
+String qaStreamedBody(
```

Comment:
The only linting rule should be 120 characters line and default android studio rules

![CleanShot 2024-06-21 at 16 25 22@2x](https://github.com/BasedHardware/Friend/assets/18078879/22e252db-5dcc-4737-a55e-39d2b3389061)

Please relint

Reply:
> This would show all ble available devices right? as far as I understand, if both options work, why to do it like this is what I don't understand

I answered this above but I want to make sure this is clear, so copy pasting the response here again:

> No it will not show all BLE devices. If the device is named different from "Friend" it will not show up. When that line is active it is filtering BLE devices by name. It will go through the onboarding screen with a real device as long as its name is "Friend" but not with a simulator. Recently we stopped filtering devices by name and use UUID instead. So this line is not only unnecessary but also prevents us from pairing a simulator.
See https://github.com/BasedHardware/Friend/pull/225#issuecomment-2169764690

Reply:
re: linting

I will apply your listing rules which are less exhaustive than my dart linter

Reply:
I wanted to add, Dart suggests we should not use more than 80 characters per line

https://dart.dev/tools/linter-rules/lines_longer_than_80_chars#:~:text=Avoid%20lines%20longer%20than%2080,0.